edx_toggles/tests/test_toggles.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+from django.test import TestCase
+from edx_toggles import toggles
+
+
+class DjangoSettingToggleTests(TestCase):
+    def test_toggle_for_absent_setting(self):
+        toggle1 = toggles.DjangoSettingToggle("NAME1", True)
+        toggle2 = toggles.DjangoSettingToggle("NAME1", False)
+
+        self.assertTrue(toggle1.is_enabled())
+        self.assertFalse(toggle2.is_enabled())
+
+    def test_toggle_for_present_setting(self):
+        toggle1 = toggles.DjangoSettingToggle("NAME1", True)
+        toggle2 = toggles.DjangoSettingToggle("NAME1", False)
+
+        with self.settings(NAME1=42):
+            self.assertTrue(toggle1.is_enabled())
+            self.assertTrue(toggle2.is_enabled())
+
+    def test_is_enabled_is_bool(self):
+        toggle1 = toggles.DjangoSettingToggle("NAME1", 42)
+        self.assertIs(True, toggle1.is_enabled())
+    
+
+class DjangoSettingDictToggleTests(TestCase):
+    def test_toggle_for_absent_setting(self):
+        toggle1 = toggles.DjangoSettingDictToggle("NAME1", "key1", True)
+        self.assertTrue(toggle1.is_enabled())
+
+    def test_toggle_for_present_setting(self):
+        toggle1 = toggles.DjangoSettingDictToggle("NAME1", "key1", True)
```

Comment:
For completeness, you could add a "key2" and test the setting (dict) being set, but the key not being set.

Reply:
:ok_hand: